templates/example_mixing_score.ipynb

```diff
@@ -0,0 +1,360 @@
+{
```

Comment:
<p>what's a cold threshold? What's an infiltrating population? What's a reference population?</p><p> </p><p>If it's symmetrical, do these matter?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>